lib/ember-orbit/schema.js

```diff
@@ -151,13 +151,13 @@ var Schema = Ember.Object.extend({
           var rel = record.__rel[link] = {};
           linkValue.forEach(function(id) {
             if (typeof id === 'object') {
-              id = id.primaryId;
+              id = id.get('primaryId');
             }
             rel[id] = true;
           });
 
         } else if (typeof linkValue === 'object') {
-          record.__rel[link] = linkValue.primaryId;
+          record.__rel[link] = linkValue.get('primaryId');
```

Comment:
Ah, that explains why I see it a lot in ember core, good to know. Good point about the pojos as well, thanks!